libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -369,9 +370,15 @@ def write_ldscript(fname):
     ram0  : org = 0x20000000, len = %uk
 }
 
-INCLUDE ../../libraries/AP_HAL_ChibiOS/hwdef/common/common.ld
+INCLUDE common.ld
 ''' % (flash_base, flash_length, ram_size))
 
+def copy_common_linkerscript(self, hwdef):
```

Comment:
Shouldn't this be receiving `outdir`? Also, the self is confusing, this doesn't belong to any class right?

Reply:
Gah.